cross_list_courses/templates/cross_list_courses/partials/list.html

```diff
@@ -0,0 +1,30 @@
+{% verbatim %}
+
+<nav>
+    {% endverbatim %}
+    <h3><a href="{% url 'dashboard_account' %}">Admin Console</a>
+        <small><i class="fa fa-chevron-right"></i></small> Cross Listing Single Pair of Courses</h3>
+    {% verbatim %}
+</nav>
```

Comment:
nitpicky stuff -- the indentations here are different from the rest of the file. I prefer two spaces (as below) but I don't think we have a standard; just would be good to stay consistent at least within a single file.